### PR TITLE
zh-cn: init the translation of `Document.children`

### PR DESCRIPTION
**Pull request by cnbilinyj**
Translate "/files/zh-cn/web/api/document/children/index.md" into Chinese

### CODE REVIEW

File: files/zh-cn/web/api/document/children/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## 规范
+
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 另请参见
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/document/children/index.md:41]:
```suggestion
## 参见
```

File: files/zh-cn/web/api/document/children/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: "Document: children 属性"
+short-title: children
+slug: Web/API/Document/children
+page-type: web-api-instance-property
+browser-compat: api.Document.children
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/document/children/index.md:6]:
```suggestion
title: Document：children 属性
slug: Web/API/Document/children
l10n:
  sourceCommit: 41a8b9c9832359d445d136b6d7a8a28737badc6b
```

参考 https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md?plain=1#L9-L25

File: files/zh-cn/web/api/document/children/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+---
+
+{{ APIRef("DOM") }}
+
+只读的 **`children`** 属性返回一个实时的 {{domxref("HTMLCollection")}}，其中包含对调用该属性的文档的所有子 {{domxref("Element", "元素")}}。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/document/children/index.md:11]:
```suggestion
**`children`** 只读属性返回一个实时的 {{domxref("HTMLCollection")}}，其中包含对调用该属性的文档的所有子{{domxref("Element", "元素", "", 1)}}。
```

**Review by yin1999 — APPROVED** [review]
Thank you @cnbilinyj, @skyclouds2001

**Pull request merged**